<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QMainWindow" name="GameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1500</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#GameWindow {
	border-image: url(:/background/desk.jpg);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius: 15px;
	background-color: rgb(27, 154, 247);
    border-color: rgb(27, 154, 247);
    color: #FFF;
    font-size: 28px;
    height: 40px;
    line-height: 40px;
    padding: 0 30px;
	font-weight: 400;
	font-family: &quot;华文楷书&quot;;
	text-decoration: none;
	text-align: center;
	margin: 5px;
}
QPushButton:visited {
    color: #FFF;
}
QPushButton:hover, QPushButton:focus {
    background-color: #4cb0f9;
    border-color: #4cb0f9;
    color: #FFF;
	text-decoration: none;
    outline: none;
}

QPushButton.disabled, QPushButton.is-disabled, QPushButton:disabled {
    top: 0 !important;
    background: #EEE !important;
    border: 1px solid #DDD !important;
    color: #CCC !important;
    opacity: .8 !important;
}
</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="playButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>出牌</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="notOutButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>不出</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1500</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
